### PR TITLE
Bump third-party libraries

### PR DESCRIPTION
**Pull request by NiLuJe**
* FBInk 1.9.1 (see note about the H2O²)
* HarfBuzz 2.1.1
* LodePNG
* LuaJIT
* NanoSVG
* OpenSSH 7.9p1
* SQLite 3.25.3

### CODE REVIEW

**Comment by NiLuJe** [comment]
As usual, don't squash ;).

**Comment by NiLuJe** [comment]
Hmm, what kind of stunningly old GCC version is the PB TC using? because I may have to burn it with fire...

File: Makefile [hunk 1]
```diff
@@ -41,7 +41,7 @@ ifndef KODEBUG
 		$(if $(or $(DARWIN),$(WIN32),$(ANDROID),$(UBUNTUTOUCH),$(APPIMAGE)),,$(OUTPUT_DIR)/dropbear) \
 		$(if $(or $(KINDLE),$(KOBO)),$(OUTPUT_DIR)/sftp-server,) \
 		$(if $(or $(KINDLE),$(KOBO)),$(OUTPUT_DIR)/scp,) \
-		$(if $(or $(KINDLE),$(KOBO)),$(OUTPUT_DIR)/fbink,) \
+		$(if $(or $(KINDLE),$(KOBO),$(CERVANTES)),$(OUTPUT_DIR)/fbink,) \
```
Comment (pazos) [inline Makefile]: that's weird, is your tree updated? In master we actually build fbink and sftp-server for cervantes: https://github.com/koreader/koreader-base/blob/master/Makefile#L30-L32

**Comment by Frenzie** [comment]
The PB TC doesn't seem to have been updated in 4 years, btw. At least not on GH.

File: Makefile (hunk 1 above)
Comment (NiLuJe) [reply Makefile]: It was indeed building, that's the strip step on !DEBUG builds ;).

**Comment by NiLuJe** [comment]
This looks to have newer stuff for newer devices, FWIW: https://github.com/blchinezu/pocketbook-sdk

**Comment by Frenzie** [comment]
Huh. We get it from https://github.com/pocketbook-free/SDK_481

File: Makefile (hunk 1 above)
Comment (pazos) [reply Makefile]: Oh, cool. Could you strip sftp-server too for me? or do I need to open a new PR?

**Comment by NiLuJe** [comment]
Then, no, yeah, it's fantastically old in there too :D.

**Comment by Frenzie** [comment]
But yeah, looks to be the same thing, just copied by someone from the official source.

**Comment by NiLuJe** [comment]
Yup, false alert ;).

**Comment by NiLuJe** [comment]
Mmmmkay, AFAICT, will probably be fixed in the next HB release, which should come Soon!

**Pull request merged**